test/e2e/registration_service_test.go

```diff
@@ -0,0 +1,53 @@
+package e2e
+
+import (
+	"crypto/tls"
+	"net/http"
+	"testing"
+	"time"
+
+	"github.com/codeready-toolchain/api/pkg/apis/toolchain/v1alpha1"
+	"github.com/codeready-toolchain/toolchain-e2e/doubles"
+	"github.com/codeready-toolchain/toolchain-e2e/wait"
+
+	framework "github.com/operator-framework/operator-sdk/pkg/test"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+)
+
+func TestRegistrationService(t *testing.T) {
+	suite.Run(t, &registrationServiceTestSuite{})
+}
+
+type registrationServiceTestSuite struct {
+	suite.Suite
+	namespace  string
+	route      string
+	testCtx    *framework.TestCtx
+	awaitility *wait.Awaitility
+}
+
+func (s *registrationServiceTestSuite) SetupSuite() {
+	userSignupList := &v1alpha1.UserSignupList{}
+	s.testCtx, s.awaitility = doubles.WaitForDeployments(s.T(), userSignupList)
+	s.namespace = s.awaitility.RegistrationServiceNs
+	s.route = s.awaitility.RegistrationServiceURL
+}
+
+func (s *registrationServiceTestSuite) TestSmokeTest() {
```

Comment:
`TestSmokeTest()` is too generic IMO.
```suggestion
func (s *registrationServiceTestSuite) TestLandingPageReachable() {
```